core/src/main/java/org/lflang/ast/ToLf.java

```diff
@@ -639,10 +640,10 @@ public MalleableString caseReaction(Reaction object) {
                               : doSwitch(varRef))
                   .collect(new Joiner(", ")));
     }
-    if (object.getName() != null) msb.append(" named ").append(object.getName());
     if (object.getCode() != null) msb.append(" ").append(doSwitch(object.getCode()));
     if (object.getStp() != null) msb.append(" ").append(doSwitch(object.getStp()));
     if (object.getDeadline() != null) msb.append(" ").append(doSwitch(object.getDeadline()));
+    if (object.isDelimited()) msb.append(";");
```

Comment:
Based on the implementation of `caseReactor`, the connections are always placed before the reactions in a reactor, so we will never have the ambiguity. Therefore, it is safe to omit the semicolon from the formatter output and to remove the `isDelimited` field from the ECore model.

Reply:
I will change the formatter to not add semicolons, but we cannot remove `isDelimited` because it is used in the validator.